README.md

```diff
@@ -494,3 +494,20 @@ Please check out common DSC Community [contributing guidelines](https://dsccommu
 
 - [CMHeartbeatDiscovery_Disabled](Source\Examples\Resources\CMHeartbeatDiscovery\CMHeartbeatDiscovery_Disabled.ps1)
 - [CMHeartbeatDiscovery_Enabled](Source\Examples\Resources\CMHeartbeatDiscovery\CMHeartbeatDiscovery_Enabled.ps1)
+
+### CMServiceConnectionPoint
```

Comment:
Need to add this under Resources at the top.

Reply:
added